app/app.go

```diff
@@ -38,7 +53,10 @@ func Run(cfg conf.Config) {
 
 	addr := fmt.Sprintf("%s:%d", config.Interface, config.Port)
 	a := App{
-		PeerStore: peer.NewPeerStore(addr),
+		PeerStore:   peer.NewPeerStore(addr),
+		CurrentUser: getCurrentUser(),
+		Chain:       getLocalChain(),
```

Comment:
This is not great. In the case when the chain is not yet local, we don't exactly want to kick off a torrenting at this exact moment. We might want the chain download to be voluntary. A CLI request 🤔